packages/consumption/test/modules/requests/OutgoingRequestsController.test.ts

```diff
@@ -213,6 +213,11 @@ describe("OutgoingRequestsController", function () {
             await Then.eventHasBeenPublished(OutgoingRequestCreatedEvent);
         });
 
+        test("sets deletionInfo in case of DeleteAttributeRequestItems", async function () {
+            await When.iCreateAnOutgoingRequestWithDeleteAttributeRequestItems();
+            await Then.theDeletionInfoOfTheAssociatedAttributesIsSet();
+        });
+
```

Comment:
This behaviour may not be desired, see other comment in `OutgoingRequestsController.ts`.